### ISSUE TITLE
activating schema based on answer from another schema

### ISSUE THREAD

**Issue by isspek** [davidjurgens/potato#71]
Thanks for implementing this awesome tool! 
I would like to annotate a sample by using with multiple schemas as the example in your [documentation](https://potato-annotation.readthedocs.io/en/latest/schemas_and_templates/#tasks-with-multiple-schemas).  In the questions of the task, one question will be depend on the answer of previous question. Would it way to disable answering questions based on conditions as I describe?

**Comment by Jiaxin-Pei** [comment]
Hi @isspek, thanks a lot for asking.
Unfortunately we don't currently support this feature. Could you say more about your specific condition? I'm happy to take a look if it's not too complicated. I might not have the time to develop this for you, but at least I could share more about where to edit!

**Comment by isspek** [comment]
Hi @Jiaxin-Pei ,

Sure. I am giving a made up example:

Text: Today is a terrible day.

Question 1:
Does this text have a property A?
A1 Yes completely
A2 Partially
A3 No

Question 2:
Does this text have a property B?
A1 Yes completely
A2 Partially
A3 No

Rewrite this text (activate it if Question 1 or Question 2 are selected as No, otherwise disable this input )

**Comment by Jiaxin-Pei** [comment]
Hi @isspek, 

For this specific case, I think the easiest solution is to directly use the textbox feature within the [radio/multiselect schema](https://potato-annotation.readthedocs.io/en/latest/schemas_and_templates/#multiple-choice). Something looks like this:
<img width="566" alt="image" src="https://github.com/davidjurgens/potato/assets/42936410/b6714679-9ba1-4f7d-baeb-af4c3eb6f7e5">

You could make NO the last option and put "Please rewrite" and the textbox below it.


If you really want to dynamically enable/disable the textbox, a quick solution would be using the customized html template.

Basically each annotation page potato is a html file built based on base templates and configurations, the back-end server render the base templates based on the input data and the configuration file when potato is launched. In you specific case, you could add a JS function in the base template which tracks the user input for question 1 and 2 and enable/disable the textbox base on the inputs. You could check the [discussion here ](https://github.com/davidjurgens/potato/issues/69)on how to use customized html templates.

Let me know if you have any questions!
